evm/utils.js

```diff
@@ -938,8 +938,10 @@ function validateGasOptions(gasOptions) {
     }
 }
 
-function isValidChain(config, chainName) {
-    const chains = config.chains;
+function isValidChain(chains, chainName) {
```

Comment:
function name is a misnomer since it doesn't return a bool
```suggestion
function validateChain(chains, chainName) {
```